.github/workflows/build.yml

```diff
@@ -21,9 +21,13 @@ jobs:
       TEST_RESULT_DIR: drop\testresults
     steps:
       - uses: actions/checkout@v1
+      - name: Setup dotnet
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '9.0.x'
       - name: Build Everything
         shell: powershell
-        run: ./build.ps1
+        run: dotnet build code.sln
```

Comment:
I have tested building the whole solution using `dotnet`, and it works. The same change will be implemented in `service-fabric-services-and-actors-dotnet`.